src/status_im/contexts/wallet/send/select_asset/view.cljs

```diff
@@ -44,34 +43,48 @@
       :on-scroll-to-index-failed    identity
       :render-fn                    asset-component}]))
 
-(defn- tab-view
-  [search-text selected-tab]
-  (case selected-tab
-    :tab/assets       [asset-list search-text]
-    :tab/collectibles [quo/empty-state
-                       {:title           (i18n/label :t/no-collectibles)
-                        :description     (i18n/label :t/no-collectibles-description)
-                        :placeholder?    true
-                        :container-style (style/empty-container-style (safe-area/get-top))}]))
-
 (defn- search-input
+  [search-text on-change-text]
+  [rn/view {:style style/search-input-container}
+   [quo/input
+    {:small?         true
+     :placeholder    (i18n/label :t/search-assets)
+     :icon-name      :i/search
+     :value          search-text
+     :on-change-text on-change-text}]])
+
+(defn collectibles-grid
   [search-text]
-  (let [on-change-text #(reset! search-text %)]
-    (fn []
-      [rn/view {:style style/search-input-container}
-       [quo/input
-        {:small?         true
-         :placeholder    (i18n/label :t/search-assets)
-         :icon-name      :i/search
-         :value          @search-text
-         :on-change-text on-change-text}]])))
+  (let [collectibles (rf/sub [:wallet/current-viewing-account-collectibles-filtered search-text])]
+    [collectibles-tab/view
+     {:collectibles         collectibles
+      :filtered?            true
+      :on-collectible-press (fn [collectible-id]
+                              (js/alert (str "Collectible to send: \n"
+                                             collectible-id
+                                             "\nNavigation not implemented yet")))}]))
```

Comment:
Temp callback, I'll solve it in the following PR, which is show the details of the collectible to send.